prisma/schema.prisma

```diff
@@ -146,7 +146,7 @@ model Project {
   lightning_address   String?
   lnurl_callback_url  String?
   tagline             String       @default("")
-  launch_status       String       @default("")
+  launch_status       String       @default("Launched")
```

Comment:
@Dolu89 this could be for a future update to the schema but I just thought that we might want the ability to hide launched projects. as a way to better curate the directory (e.g. someone reports the project as not suitable or something)

Reply:
Interesting point.
Yes, at one point we will probably have some projects that we want to hide/delete.

But what I'm thinking is that this should be something different that the `launch_status` field

Because `launch_status` represents the launch status, but the state: "hidden" for example isn't a launch status.

So for that we might add a new field like: "is_hidden" maybe...

Reply:
I agree with @MTG2000 👍